modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -57,24 +57,24 @@ private bool CreateProjectSolution()
             {
                 pr.Step("Generating C# project...".TTR());
 
-                string resourceDir = ProjectSettings.GlobalizePath("res://");
-
-                string path = resourceDir;
+                GodotSharpDirs.RefreshProjectLocation();
```

Comment:
This should not be called manually. Instead, it should be called when the `EditorPlugin.ProjectSettingsChanged` signal is emitted (`GodotSharpEditor` inherits `EditorPlugin`).

Reply:
I connected it with the `EditorPlugin.ProjectSettingsChanged`. Should I also need to disconnect when the plugin is disabled? Since we didn't override the `_DisablePlugin`, I'm not very sure if it is necessary.

Reply:
OK! Thank you for your explaining! 😄